docs/user/coordinates.md

```diff
@@ -0,0 +1,43 @@
+<!--Copyright (C) SuperDARN Canada, University of Saskatchewan 
+Author(s): Marina Schmidt 
+Modifications:
+
+Disclaimer:
+pyDARN is under the LGPL v3 license found in the root directory LICENSE.md 
+Everyone is permitted to copy and distribute verbatim copies of this license 
+document, but changing it is not allowed.
+
+This version of the GNU Lesser General Public License incorporates the terms
+and conditions of version 3 of the GNU General Public License, supplemented by
+the additional permissions listed below.
+-->
+
+# Coordinate Systems 
+---
+
+pyDARN offers several different coordinate systems for plotting: 
+
+## Range-Time Plots 
+
+**Range Gates**: `Coords.RANGE_GATE` a rectangle section determined by beam width and set distance for each range (nominally 45 km). RAWACF and FITACF data give their parameter values with respect to range gates  
+
+**Slant Range**: `Coords.SLANT_RANGE` is a conversion from range gates to km units.  
```

Comment:
```suggestion
**Slant Range**: `Coords.SLANT_RANGE` is a conversion from range gates to km units.  Slant range estimates the distance of ionospheric echos from the radar, using the time it takes for the radio wave to travel to the ionosphere and return, assuming the radio wave is travelling at the speed of light.
```

Reply:
Do we want to add a warning with rxrise and frang?

Reply:
Yeah, I'm just looking over those issues I made to write it properly

Reply:
I've amended the text to include a run down of things that are of note in how we calculate slant range, but that it's outside our remit to fix these and were essentially waiting it out until it's fixed at a higher level.

I don't know if the formatting is right in the suggestion though, so it might need going in and fixing?